trlx/orchestrator/ppo_orchestrator.py

```diff
@@ -40,6 +40,7 @@ def __init__(
 
         if not hasattr(self.trainer.model, "frozen_head"):
             self.ref_model = self.trainer.get_arch(self.trainer.config)
+            self.ref_model.to(self.trainer.accelerator.device)
```

Comment:
@Dahoas

Reply:
Have we verified this works? I recall accelerate freezing up if I started putting multiple models on gpu(though this could've just been the sentiment pipeline we were using for sentiments task)

Reply:
@Dahoas  yes that works. 
e.g. https://wandb.ai/pvduy/trlx/runs/2wgpt4im

Reply:
I do agree with the larger model we should distribute multiple models on multiple gpus, but for this, I think we should keep it on GPUs rather than CPUs, they are super slow.